apps/src/templates/manageStudents/ManageStudentsFamilyNameCell.jsx

```diff
@@ -41,6 +41,7 @@ class ManageStudentFamilyNameCell extends Component {
           <div>
             <span data-for={tooltipId} data-tip>
               <input
+                id="uitest-family-name"
```

Comment:
i think this will be a duplicate id when you try to "edit all" on this table, won't it?

Reply:
I considered ID or username, but because those are created on the spot I'm not convinced those are easy to figure out. I'll investigate passing through some kind of row number.

Reply:
Update: from what I can read of 'name', it doesn't have to be unique the way 'id' does (can be used to group input values together across forms). I've switched to this approach but please flag if that doesn't feel right either!

Reply:
name should be fine, since i think this is submitted via ajax and not a web form. if this had been a div or something that didn't support `name`, something like `data-uitest="family-name"` would also not break anything and give you something to select by in the tests.

Reply:
That makes sense to me- thanks! Resolving.